docs/api/php_api/php_api.md

```diff
@@ -73,6 +73,14 @@ These objects provide you with lower-level information.
 For instance, `ContentInfo` contains `currentVersionNo` or `remoteId`,
 while `Content` enables you to retrieve Fields, Content Type, or previous versions.
 
+!!! note
+
+    From v3.0 the Public API value objects should not be serialized.
```

Comment:
```suggestion
    The Public API value objects should not be serialized.
```
All the versions you'll be merging this to are > 3.0, so you can drop naming the version (only 3.3+ are in maintenance anyway)

Reply:
It was added per Adam's request.

Reply:
@adamwojs our policy for doc is that we document the current version only, without referring to "since"s or "until"s, unless it's release notes or upgrade guides.